platform/barefoot/libsaithrift-dev.mk

```diff
@@ -1,20 +1,27 @@
-# libsaithrift-dev package
-
 SAI_VER = 0.9.4
 
-LIBSAITHRIFT_DEV = libsaithrift-dev_$(SAI_VER)_amd64.deb
+LIBSAITHRIFT_DEV = libsaithrift$(SAITHRIFT_VER)-dev_$(SAI_VER)_amd64.deb
 $(LIBSAITHRIFT_DEV)_SRC_PATH = $(SRC_PATH)/sonic-sairedis/SAI
-$(LIBSAITHRIFT_DEV)_DEPENDS += $(LIBTHRIFT) $(LIBTHRIFT_DEV) $(PYTHON_THRIFT) $(THRIFT_COMPILER) $(BFN_SAI) $(BFN_SAI_DEV)
-$(LIBSAITHRIFT_DEV)_RDEPENDS += $(LIBTHRIFT) $(BFN_SAI)
+ifeq ($(SAITHRIFT_V2),y)
+$(LIBSAITHRIFT_DEV)_DEPENDS += $(LIBTHRIFT_0_13_0) $(LIBTHRIFT_DEV_0_13_0) $(PYTHON3_THRIFT_0_13_0) $(THRIFT_COMPILER_0_13_0)
+$(LIBSAITHRIFT_DEV)_RDEPENDS += $(LIBTHRIFT_0_13_0)
+$(LIBSAITHRIFT_DEV)_BUILD_ENV = SAITHRIFTV2=true SAITHRIFT_VER=v2
+else
+$(LIBSAITHRIFT_DEV)_DEPENDS += $(LIBTHRIFT) $(LIBTHRIFT_DEV) $(PYTHON_THRIFT) $(THRIFT_COMPILER)
+$(LIBSAITHRIFT_DEV)_RDEPENDS += $(LIBTHRIFT)
+endif
```

Comment:
There is no need to add this if statement. BFN SDK can't work with Thrift 0.11.0. Even more. Once this PR will be merged https://github.com/Azure/sonic-buildimage/pull/9870 BFN SDK becomes dependent on thrift 0.14.1 to avoid run time issues. So I have created https://github.com/Azure/sonic-buildimage/pull/9881 to bump the thrift version to 0.14.1.
Also, there is PR https://github.com/Azure/sonic-buildimage/pull/9884 with further changes for BFN needed for syncd-rpc.

Reply:
ok, how about the old test case in the previous version, will the 202012 branch also be shipped with thrift_14?

Reply:
As I know not. The SDK for 202012 is not dependent on thrft_14, so there are no plans to cherry-pick thrift 14 changes to older releases. But I will double-check this.

Reply:
updated with 0.14